packages/calcite-components/src/components/tip/tip.scss

```diff
@@ -1,23 +1,44 @@
+/**
+* CSS Custom Properties  
+*
+* These properties can be overridden using the component's tag as selector.
+*
+* @prop --calcite-tip-border-color: The border color of the component.
+* @prop --calcite-tip-color: The background color of the component.
+* @prop --calcite-tip-font-size: The font size of the component.
+* @prop --calcite-tip-text-color: The text color of the component.
+*
+* @prop --calcite-internal-tip-padding: The paddings around component elements.
```

Comment:
Internal bars should be in a separate comment block so they are not publicly documented.